Dockerfile

```diff
@@ -13,7 +13,7 @@ ENV LANGUAGE en_US:en
 ENV LC_ALL en_US.UTF-8
 
 WORKDIR /app
-ADD requirements/requirements.txt /app/requirements/requirements.txt
+ADD requirements/base.txt /app/requirements/base.txt
```

Comment:
I think this is pointless because of the later `ADD . /app` but hey, consistency.

Reply:
Adding it here means we wont reinstall dependencies on subsequent docker builds unless the requirements have actually changed.

Reply:
It is useful when doing dev for a lot of apps

Reply:
Ah! Makes sense. :-)